sdk/storage/storage-file-share/.vscode/launch.json

```diff
@@ -34,7 +34,7 @@
         "${workspaceFolder}/test/node/*.spec.ts"
       ],
       "env": { "TS_NODE_COMPILER_OPTIONS": "{\"module\": \"commonjs\"}" },
-      "envFile": "${workspaceFolder}/../.env",
```

Comment:
Why change this?

Reply:
We already changed it to use local .env in unit test. Just to make it all consistent.